.woke.yml

```diff
@@ -0,0 +1,4 @@
+rules:
+  # Ignore "whitelist" and "master".
+  - name: whitelist
+  - name: master
```

Comment:
Out of curiosity, are these words included in this repository?

Reply:
Yes, they are